docs/README.md

```diff
@@ -235,7 +235,7 @@ Bireylerin internet ortamında daha güvenli bir deneyim yaşamalarını sağlam
     <div class="service-box">
         <img src="docs/images/keepassxc.png" alt="icon">
         <div class="title">
-            <a href="https://guvendekal.org/#/sifre-yoneticileri?id=-keepassxc" target="_blank" style="color: inherit; text-decoration: none;">KeepassXC</a>
+            <a href="https://guvendekal.org/#/sifre-yoneticileri?id=-keepassxc" target="_blank" style="color: inherit; text-decoration: none;">Keepass</a>
```

Comment:
Önerdiğimiz uygulama Keepas değil KeepassXC aslında. KeepassXC, Keepass'ın çatalı ve Keepass'a göre Android, Linux gibi farklı platformlarda uygulama yayınlıyor.

Reply:
**KeepassXC** sadece Windows, Mac ve Linux platformlarında mevcut ama Keepass'in **KeepassDX** isminde ayrı bir Android uygulaması bulunuyor. Sanırım geliştiricileri KeepassXC'den bağımsız. Bu nedenle Keepass olarak belirtmemizin daha doğru olacağını düşünüyorum.

Reply:
Evet yanlış hatırlıyormuşum ancak platform desteğinden ötürü KeepassXC'i eklemiştik diye hatırlıyorum, [Şifre Yöneticileri](https://guvendekal.org/#/sifre-yoneticileri?id=-keepassxc) sayfasında da KeepassXC şeklinde yazdık ve KeepassXC'in logosunu kullandık.

Reply:
Çoğu yerde Türkçe kullanmak daha doğru ve kullanmak gerekli diye düşünüyorum ancak genel ağ ifadesi tam olarak internet kelimesini karşılayamayabilir evet.